cogs/send_introduction_reminders.py

```diff
@@ -182,13 +182,16 @@ class OptOutIntroductionRemindersView(View):
         This discord.View contains a single button that can change the state of whether the
         member will be sent reminders to send an introduction message in
         your group's Discord guild.
-        The view object will be sent to the member's DMs, after a delay period after
+        The view object will be sent to the member's DMs after a delay period after
         joining your group's Discord guild.
         """
 
         @override
         def __init__(self, bot: TeXBot) -> None:
-            """Initialize a new discord.View, to opt-in/out of introduction reminders."""
+            """Initialise a new discord.View, to opt-in/out of introduction reminders."""
+            # NOTE: The attribute/variable name `bot` is used here for consistency.
+            # NOTE: `tex_bot` would be preferred but would be inconsitent with the required attribute name of Pycord's context classes
+            # NOTE: See https://github.com/CSSUoB/TeX-Bot-Py-V2/issues/261
```

Comment:
Perhaps just the link to the issue? And remove the additional text. I would just love to have some sort of comment in the code about why `bot` is used, so that we don't go through the same whole process again in years time of attempting to rename the variables for consistency, the realising it's not possible.

Reply:
I think this would be a lot neater if the link is also just on the same line as the code rather than above.

Reply:
Maybe not even the link but just a note saying the variable *has* to be named `bot` and to see issue number whatever for more details

Reply:
Just a floating issue number isn't very helpful IMO. What happens if we move repository? Makes it harder to connect threads of information. I don't mind not including the note text in the code.

So just:
```python
# See https://github.com/CSSUoB/TeX-Bot-Py-V2/issues/261
```

Reply:
honestly I don't like any of the solutions, I'd rather just leave it without anything there tbh, but if you're dead set on it the link should be fine.

Reply:
I'm not dead set on that specifically. Just some good way of ensuring a comment is there & I thought the link would be good because we don't have to write out all the details. It would be explained after clicking the link.

I might mull over this and think about if there is a better way to describe what is happening without cluttering up the code. Any better thoughts would also be appreciated.

Reply:
I couldn't think of anything better than the link.